README.md

```diff
@@ -1,2 +1,171 @@
-# budget-buddy
-Budget Buddy is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+# Recipe App
+
+In this project, you will start building a Recipe app that keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe.
+
+# 📗 Table of Contents
+
+- [Recipe App](#budget-buddy)
+  - [Getting Started](#getting-started)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 budget-buddy](#budget-buddy)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Demo Link](#live-demo)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👤 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 📖 budget-buddy <a name="about-project"></a>
+
+The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.rubyguides.com">Ruby</a></li>
+    <li><a href="https://www.postgresql.org/">Postgresql</a></li>
+    <li><a href="https://guides.rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+
+## Project Description <a name="link-preview"></a>
+
+- [Project description](https://www.loom.com/share/d57b42eddeac46968bcc56fd83afce4f)
+
```

Comment:
- [ ] It's great to mention the feature you are proud of implementing, but it would be nice after explaining the quality feature, you show the highlighted code that you wrote to achieve that feature. This will complete the requirement as expected.
- [ ] After adding the category selection feature for creating a transaction, kindly redo the video so all features are present in your video presentation. That will add more value to your app's demo and serve as a reference for potential employers looking at your repo. 👍🏽